Notify/Notify/Notify.Android/Services/GoogleMapsService.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Android.Content;
+using Android.Content.PM;
+using Geolocation;
+using Notify.Azure.HttpClient;
+using Notify.Core;
+using Notify.Services;
+using Xamarin.Essentials;
+using Location = Xamarin.Essentials.Location;
+
+namespace Notify.Droid.Services
+{
+    public class GoogleMapsService : ExternalMapsService
+    {
+        private readonly LoggerService r_Logger = LoggerService.Instance;
+        private readonly Context r_Context;
+
+        private GoogleMapsService(Context context)
+        {
+            r_Context = context;
+        }
+
+        public static ExternalMapsService Initialize(Context context)
+        {
+            if (m_Instance == null)
+            {
+                m_Instance = new GoogleMapsService(context);
+            }
+    
+            return m_Instance;
+        }
+
+        public static GoogleMapsService GetInstance()
+        {
+            if (m_Instance == null)
+            {
+                throw new InvalidOperationException("MyMapHandler has not been initialized. Call Initialize first.");
+            }
+            
+            return (GoogleMapsService)m_Instance;
+        }
+
+        public void OpenGoogleMapsNavigation(double latitude, double longitude)
+        {
+            //if (IsGoogleMapsInstalled()) - TODO  - uncomment this line and check on physical device
+            if(true)
+            {
+                var uri = Android.Net.Uri.Parse($"google.navigation:q={latitude},{longitude}");
+                var intent = new Intent(Intent.ActionView, uri);
```

Comment:
1. Declare variables in the beginning of the method.
2. Use explicit types instead of var.